### PR TITLE
Update list of papers using AMICI

### LINKED ISSUE #1504: New publications using amici
https://www.sciencedirect.com/science/article/pii/S1096717621000161

https://www.biorxiv.org/content/10.1101/2021.03.25.437053v1.abstract

https://research.wur.nl/en/publications/model-driven-design-of-mycoplasma-as-a-vaccine-chassis

### PR DESCRIPTION
**Pull request by dweindl**
Closes #1504

### CODE REVIEW

File: documentation/amici_refs.bib [hunk 1]
```diff
@@ -811,6 +811,59 @@ @Article{RaimundezDud2021
...
+  address = {Wageningen},
+  comment = {WU thesis 7758 Includes bibliographical references. - With summaries in English, Italian and Spanish
+10.18174/539593},
+  doi     = {10.18174/539593},
+  issn    = {9789463956864},
+  url     = {https://edepot.wur.nl/539593},
+}
+
+@Article{VanhoeferMat2021,
+  author    = {Jakob Vanhoefer and Marta R. a. Matos and Dilan Pathirana and Yannik Schälte and Jan Hasenauer},
```
Comment (dilpath) [inline documentation/amici_refs.bib:855]:
```suggestion
  author    = {Jakob Vanhoefer and Marta R. A. Matos and Dilan Pathirana and Yannik Schälte and Jan Hasenauer},
```

File: documentation/references.md [hunk 2]
```diff
@@ -15,13 +18,22 @@ If you applied AMICI in your work and your publication is missing, please let us
 <div id="ref-StaedterSch2021">
 <p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2021. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>Scientific Reports</em> 11 (1): 2696. <a href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.</p>
 </div>
+<div id="ref-StenPod2021">
+<p>Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder, Maria Engström, and Gunnar Cedersund. 2021. “A Multi-Data Based Quantitative Model for the Neurovascular Coupling in the Brain.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.</p>
+</div>
+<div id="ref-VanhoeferMat2021">
+<p>Vanhoefer, Jakob, Marta R. a. Matos, Dilan Pathirana, Yannik Schälte, and Jan Hasenauer. 2021. “Yaml2sbml: Human-Readable and -Writable Specification of Ode Models and Their Conversion to Sbml.” <em>Journal of Open Source Software</em> 6 (61): 3215. <a href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.</p>
```
Comment (dilpath) [inline documentation/references.md:25]:
```suggestion
<p>Vanhoefer, Jakob, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte, and Jan Hasenauer. 2021. “yaml2sbml: Human-readable and -writable specification of ODE models and their conversion to SBML.” <em>Journal of Open Source Software</em> 6 (61): 3215. <a href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.</p>
```

Comment (dilpath) [reply documentation/references.md:25]: Is this generated automatically from the `.bib`? There, the title is capitalized as in the publication.

File: documentation/references.md [hunk 3]
```diff
@@ -15,13 +18,22 @@ If you applied AMICI in your work and your publication is missing, please let us
 <div id="ref-StaedterSch2021">
 <p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2021. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>Scientific Reports</em> 11 (1): 2696. <a href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.</p>
 </div>
+<div id="ref-StenPod2021">
+<p>Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder, Maria Engström, and Gunnar Cedersund. 2021. “A Multi-Data Based Quantitative Model for the Neurovascular Coupling in the Brain.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.</p>
```
Comment (dilpath) [inline documentation/references.md:22]: Is it intentional that many publications appear to have the first author as `[Last Name], [First Name]` then subsequent authors as `[First Name] [Last Name]`?

**Review by dilpath — APPROVED** [review]

File: documentation/references.md (hunk 2 above)
Comment (dweindl) [reply documentation/references.md:25]: Yes, it's auto-generated. With the current solution, it's unfortunately not easily possible to change the style. Maybe needs to be rewritten at some point.

**Comment by FFroehlich** [comment]
How about also adding https://arxiv.org/abs/2105.12008 and https://www.biorxiv.org/content/10.1101/2021.05.20.445065v1.abstract ?

**Review by FFroehlich — APPROVED** [review]

File: documentation/references.md (hunk 3 above)
Comment (dweindl) [reply documentation/references.md:22]: Not my favorite, but that's what pandoc-citeproc does :)

**Pull request merged**